migrations/sql/R__view_now_application_gis_export.sql

```diff
@@ -1,3 +1,17 @@
+-- Creation of these two NRIS schema tables allows this migration to succeed locally.
+CREATE SCHEMA IF NOT EXISTS nris;
```

Comment:
This doesn't impact the NRIS container's ability to create the schema itself with it's code first migrations?

Reply:
![image](https://user-images.githubusercontent.com/17113094/122318329-b846c880-ced3-11eb-9bb1-b7d8f0969575.png)